dashboard/db/migrate/20170612195103_change_plc_course_units_unit_description.rb

```diff
@@ -0,0 +1,14 @@
+class ChangePlcCourseUnitsUnitDescription < ActiveRecord::Migration[5.0]
+  def change
+    reversible do |dir|
+      change_table :plc_course_units do |t|
+        dir.up do
+          t.change :unit_description, :text
+        end
+        dir.down do
+          t.change :unit_description, :string
+        end
+      end
+    end
+  end
```

Comment:
Given that no part of the `change` is reversible, it seems more succinct and simpler to simply do
```
def up
  change_table :plc_course_units do |t|
    t.change :unit_description, :text
  end
end

def down
  change_table :plc_course_units do |t|
    t.change :unit_description, :string
  end
end